src/docs/arch-column-row.dox

```diff
@@ -0,0 +1,93 @@
+/*! @arch_page arch-column-row Row Store & Column Store
+
+@section Definition
+
+A row oriented storage is an access method where all columns of a row are stored
+together, the data is stored by row, while a column oriented system stores its data tables by
+column. WiredTiger supports both and each management system is represented internally by a certain
+type of B-Tree data structure.
+
+Keys and values in row-stores can be anything of arbitrary length, thus, row-stores implement a
+generalized version of a B-Tree. Furthermore, keys are explicitly stored. Column-stores use a 64-bit
+unsigned integer record id as keys, thus, column-stores implement a specialized version of a B-Tree
+where the key has a predictable length. In the case of column-stores, keys are implied given the
+B-Tree position.
+
+There are two types of column-stores in WiredTiger:
+- The fixed length column-store, as its name implies, the value has a fixed length, and furthermore
+the value is restricted to 1 and 8 bits in length. The bit length is specified when the column store
+is created. The fixed length column store has specialized use cases like bitmaps.
+- The variable length is a more general case which allows for values to have any length similarly to
+row-stores.
+
+@section In-memory and on-disk format
+
+Row-stores and column-stores have a specific in-memory and on-disk data structure. Please refer to
+@ref arch-data-file to learn more.
+
+@section Row-stores
+
+Row-stores are probably seen as the traditional access method and they serve a general purpose. In
+WiredTiger, row-store keys are explicitly stored and they are usually duplicated within the same
+single file which directly impacts the on-disk storage size.
+
+@section Variable length column-stores
+
+The main advantage of variable length column-stores is on-disk storage size. Their keys are not
+stored but derived from the B-Tree location in the memory. In fact, there is a starting key on each
```

Comment:
`from the B-Tree location in the memory`-> `from the row's position in the B-Tree`?